### PR TITLE
Added Hacktoberfest as keyword

### LINKED ISSUE #45: [FEAT] Adding phrases for 'hacktoberfest' keyword
# Feature Request
Add responses for "hacktoberfest"

Yah, the problem is that sometimes a few other people in the hacktoberfest team feel left out.
Even though all of them are lovely. (It's like we're playing favourites)
Isn't it weird that the We-love-hacktoberfest bot doesn't have phrases for "hacktoberfest"

## Describe the solution you'd like

I shall add _hacktoberfest_ phrases to the bot and make a PR which maybe if you find satisfactory, can be used to create more love in the server for the rest of the team too

## Additional context
you um stated in the server to make a pr.
Screenshot 2020-10-22 at 11 52 45 PM

### PR DESCRIPTION
**Pull request by Vyvy-vi**
# Pull Request

Adding phrases for hacktoberfest to the bot [the rest of the team is left out from love :( ]
❤️
## Description:
Same old stuff the bot did for matt, nick, luke, daniel, raven... just with 'hacktoberfest' as keyword(or trigger-word)

## Related Issue: # 45

Closes #45

### CODE REVIEW

File: src/phrases.ts [hunk 1]
```diff
@@ -245,3 +250,63 @@ export const daniel = [
...
+  "Hacktoberfest Team is Marvellous!",
+  "Hacktoberfest Team is Wonderful!",
+  "Hacktoberfest Team is Mind-blowing!",
+  "Hacktoberfest Team is Thrilling!",
+  "Hacktoberfest Team is Fantastic!",
+  "Hacktoberfest Team is Terrific!",
+  "Hacktoberfest team always makes us feel welcomed :heart:",
+  "Hacktoberfest team is amazing",
+  "When Hacktoberfest Team says they will do something, We trust them. :heart:",
+  "Hacktoberfes team does everything so well!",
```
Comment (nhcarrigan) [inline src/phrases.ts:302]:
```suggestion
  "Hacktoberfest team does everything so well!",
```
Typo.

**Review by nhcarrigan — CHANGES_REQUESTED** [review]
One nitpick. 

Fix the linting issues as well and this should be good to go.

File: src/phrases.ts (hunk 1 above)
Comment (Vyvy-vi) [reply src/phrases.ts]: oh wupps

**Review by nhcarrigan — APPROVED** [review]
LGTM~! 🥳 

Sorry for the delay!

**Pull request merged**